examples/web-socket-events/src/main/java/com/iland/app/App.java

```diff
@@ -125,10 +126,11 @@ static void webSocketProcessEvents() {
     final SocketSubscription orgSubscription =
         webSocketClient.consumeEvents(orgEventConsumer, orgEventFilter);
 
-    // Keeping the process alive so the web socket can consume events.
-    // Note that there is no error handling for interrupted or connection errors
-    // and that you should add try/catch block to handle such cases.
-    while (true) {
+    // Keeping the process alive for 5 minutes so the web socket can consume events.
```

Comment:
the process is still going to stay alive beyond 5 minutes because this is in the while loop